package.json

```diff
@@ -117,7 +119,8 @@
     "e2e": "start-server-and-test start http://localhost:3000 test:e2e",
     "e2e-on-build": "cross-env CYPRESS_baseUrl=http://localhost:8080 start-server-and-test start:server:e2e-on-build http://localhost:8080 test:e2e",
     "badge": "cross-env NODE_CONFIG_ENV=test TRACE_SERVICES=true node scripts/badge-cli.js",
-    "build-docs": "rimraf api-docs/ && jsdoc --pedantic -c ./jsdoc.json . && echo 'contributing.shields.io' > api-docs/CNAME"
+    "build-docs": "rimraf api-docs/ && jsdoc --pedantic -c ./jsdoc.json . && echo 'contributing.shields.io' > api-docs/CNAME",
+    "migrate": "node scripts/write-migrations-config.js > migrations-config.json && node-pg-migrate --config-file=migrations-config.json"
```

Comment:
This is a bit hacky. I want the DB config to be specify-able in yaml or env vars like all our other setting but `node-pg-migrate` has its own ideas about how to handle config https://salsita.github.io/node-pg-migrate/#/cli . This little script reads the postgres connection string (however it has been set - yaml or env vat) and writes it to a config file for `node-pg-migrate`. I did consider writing a script that outputs the connection string to stdout and then setting an env var with `DATABASE_URL=$(node /scripts/get-dataabse-url.js)` but I picked this way because it should be cross-platform for Windows users.